OWLChangeOntology/.htaccess

```diff
@@ -10,40 +10,30 @@ AddType application/n-triples .n3
 AddType application/ld+json .jsonld
 # Rewrite engine setup
 RewriteEngine On
-#Change the path to the folder here
-RewriteBase /
 
 # Rewrite rule to serve JSON-LD content from the vocabulary URI if requested
 # Placed before HTML to support serving JSON-LD from a browser (e.g., JSON Playground)
 RewriteCond %{HTTP_ACCEPT} application/ld\+json
-RewriteRule ^$ ontology.jsonld [R=303,L]
+RewriteRule ^$ https://diegocondeherreros.github.io/OWLChangeOntology/ontology.jsonld [R=303,L]
+RewriteRule ^ontology\.jsonld$ https://diegocondeherreros.github.io/OWLChangeOntology/ontology.jsonld [R=303,L]
 
 # Rewrite rule to serve HTML content from the vocabulary URI if requested
 RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
 RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^$ https://github.com/DiegoCondeHerreros/OWLChangeOntology/blob/main/OWLChangeOntologyDocumentation/index-en.html [R=303,L]
-
-# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
-RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
-RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
-RewriteRule ^$ ontology.owl [R=303,L]
+RewriteRule ^$ https://diegocondeherreros.github.io/OWLChangeOntology [R=303,L]
 
 # Rewrite rule to serve N-Triples content from the vocabulary URI if requested
 RewriteCond %{HTTP_ACCEPT} application/n-triples
-RewriteRule ^$ ontology.nt [R=303,L]
+RewriteRule ^$ https://diegocondeherreros.github.io/OWLChangeOntology/ontology.nt [R=303,L]
+RewriteRule ^ontology\.nt$ https://podio.linkeddata.es/ontology.nt [R=303,L]
```

Comment:
Is this the correct target?  Or should it match the above one?

Reply:
podio is within the namespace of our group, so I think it's ok